src/components/health/health.tsx

```diff
@@ -40,6 +40,7 @@ export type EuiHealthProps = CommonProps &
      * Sets the color of the dot icon.
      * It accepts any `IconColor`: `default`, `primary`, `secondary`, `success`, `accent`, `warning`, `danger`, `text`,
      * `subdued` or `ghost`; or any valid CSS color value as a `string`
+     * **`secondary` color is DEPRECATED, use `success` instead**
```

Comment:
Lets also remove it from the list of available options
```suggestion
     * It accepts any `IconColor`: `default`, `primary`, `success`, `accent`, `warning`, `danger`, `text`,
     * `subdued` or `ghost`; or any valid CSS color value as a `string`
     * **`secondary` color is DEPRECATED, use `success` instead**
```